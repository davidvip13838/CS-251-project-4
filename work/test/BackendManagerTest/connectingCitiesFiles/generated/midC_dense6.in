500
49 0
1 3
16 35
13 23
22 24
45 47
35 32
44 17
11 34
27 44
40 37
41 8
39 50
49 44
37 49
41 26
27 24
44 15
19 50
7 36
32 27
50 6
21 35
33 1
3 30
27 22
6 35
5 35
0 12
36 28
30 11
48 37
44 14
10 17
10 6
4 29
13 4
23 43
21 23
37 48
23 11
20 12
13 0
25 14
37 44
21 28
23 45
40 47
42 32
0 29
27 9
41 32
15 36
29 18
13 7
28 28
38 25
40 4
35 24
30 23
9 37
24 45
50 4
34 41
48 40
24 41
42 22
48 5
50 1
3 25
42 28
36 25
35 8
38 23
46 43
9 43
48 49
48 23
3 27
25 13
18 49
22 12
2 18
37 5
23 7
6 19
26 16
41 37
26 38
14 33
5 10
27 21
39 41
40 32
5 17
37 36
16 50
22 11
43 42
49 5
50 40
12 36
42 29
24 10
24 20
10 50
34 32
24 34
3 35
19 36
7 21
50 24
35 45
41 9
40 26
8 37
43 47
9 25
6 14
5 34
7 37
4 25
4 15
27 36
40 21
32 3
21 45
12 49
40 20
34 11
1 38
49 6
3 19
49 48
22 27
30 25
0 32
19 6
0 33
32 16
21 13
40 2
29 6
0 43
31 13
15 34
8 47
27 28
10 39
13 36
50 43
18 1
7 20
34 36
27 42
38 36
28 38
27 20
8 29
39 16
17 36
44 16
47 36
23 9
30 21
8 30
23 39
27 6
11 50
35 17
19 9
15 27
20 47
26 17
31 17
17 31
48 46
25 42
31 47
18 11
39 11
37 21
35 49
14 23
42 25
0 48
32 24
38 9
46 21
10 19
39 35
45 16
9 20
36 20
15 47
12 48
24 11
27 19
29 24
35 50
29 15
11 21
4 16
10 2
23 50
28 20
42 17
16 34
38 16
49 2
16 44
34 27
49 20
26 22
4 17
1 30
14 0
25 15
48 1
36 36
13 8
9 47
36 14
45 9
33 20
30 49
33 49
40 35
21 38
39 25
49 42
21 14
23 29
48 41
6 25
43 37
48 9
40 42
8 15
6 12
50 18
0 15
43 2
48 4
16 30
50 21
45 10
7 41
49 31
15 2
14 26
17 3
44 37
22 50
18 46
32 11
15 37
8 11
40 30
28 19
43 19
45 42
16 20
24 44
33 22
35 38
16 25
43 25
50 41
6 43
22 38
39 47
26 2
23 24
27 27
29 9
10 41
50 5
18 13
16 47
21 10
46 23
46 26
19 5
14 24
18 17
39 46
2 5
38 14
46 33
44 1
47 26
10 16
32 30
46 44
23 40
46 34
33 13
6 36
10 22
47 27
0 1
42 7
45 32
35 31
32 41
16 18
1 2
7 17
5 11
0 46
26 14
11 27
35 35
38 31
22 14
49 10
28 18
23 28
15 46
10 0
9 12
18 29
30 1
44 27
46 28
39 36
11 33
23 17
49 41
37 19
17 6
16 1
34 17
22 37
17 34
36 48
48 15
8 26
20 43
5 46
10 20
41 33
25 34
3 1
34 16
32 1
37 32
17 38
49 37
19 35
17 26
30 34
14 36
42 8
9 28
18 40
41 24
11 9
33 30
6 8
29 50
40 41
39 10
6 28
20 29
26 31
19 38
49 40
40 18
14 22
43 44
43 20
11 47
40 36
2 28
37 10
50 25
43 38
16 12
33 44
2 1
46 8
28 27
15 13
17 42
24 38
23 38
45 31
15 5
50 34
7 32
26 50
6 45
26 49
13 37
7 14
4 7
31 0
16 31
20 27
45 34
43 35
3 7
1 31
38 33
13 44
23 8
34 50
14 32
21 4
31 5
9 49
25 46
4 34
36 19
6 21
24 14
27 1
47 43
43 45
19 44
19 17
27 8
31 4
34 25
4 31
30 24
32 28
50 17
24 43
14 9
11 45
15 6
13 14
31 26
19 20
6 22
10 4
1 6
12 37
17 41
7 34
39 49
48 7
32 2
26 18
22 45
0 10
23 41
43 0
42 2
40 25
4 13
47 37
17 21
39 4
47 0
17 46
31 3
30 14
21 36
33 32
13 2
40 45
49 30
14 6
32 4
4 23
0 34
36 26
48 30
8 48
34 33
20 1
1 8
6 30
45 45
7 29
8 23
0 6
41 6
23 49
1 48
13 32
13 16
3 18
2 50
16 5
1 13
4 41
34 10
6 49
19 3
49 26
42 20
1 20
44 40
5 41
35 19
