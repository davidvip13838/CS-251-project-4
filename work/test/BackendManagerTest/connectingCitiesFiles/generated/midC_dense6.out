785.7788209253264